board.h

```diff
@@ -0,0 +1,71 @@
+#include <vector>
+#include <iostream>
+#include <string>
+#include <cstdlib> 
+
+struct Position{
+    int xpos;
+    int ypos;

```

Comment:
#could make them `unsigned` so negative values are not even able to be represented